src/otx/algorithms/common/configs/training_base.py

```diff
@@ -231,6 +231,15 @@ class BasePostprocessing(ParameterGroup):
             affects_outcome_of=ModelLifecycle.INFERENCE,
         )
 
+        nms_iou_threshold = configurable_float(
+            default_value=0.0,
```

Comment:
Why do we have 0 here? Should it be ~0.5?

Reply:
Every detectors have their own nms iou threshold in their model.py. It is different across the architecture, So here default value 0.0 means we're going to use the default value of object detector. 
In addition, this methods can prevent confusion from OTX side. Currently OTX do not know whether hyperparameters come from defaults or users command. So this way can easily determine whether the nms iou threshold come from users or defaults confiuration.

Reply:
The problem that in Geti UI, users would see 0.0 as IoU threshold, which looks like incorrect value.